.eslintignore

```diff
@@ -0,0 +1,2 @@
+**/*.test.js
+**/*.test.jsx
```

Comment:
Can you revert this? It would be best to keep coding expectations consistent across all files.